### ISSUE TITLE
fake-tabs: allow aria-current="true" 

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/ebayui-core#1354]
Scope: allow configuration/enumeration of aria-current.

Currently, fake-tabs always sets `aria-current=page` for the current tab. However, this is not always valid in cases when there is another level of site navigation under the tabs.

In the screenshot below, both the first tab link  "Activity") and the first link of secondary navigation ("Summary') both reflect the current page url, and so it is valid for them to both have aria-current=page.

<img width="893" alt="Screen Shot 2021-03-18 at 1 40 54 PM" src="https://user-images.githubusercontent.com/38065/111696138-2c989f80-87f1-11eb-8474-d56b7c651a04.png">

However, in the following case, only the "Recently viewed" link should have aria-current=page. The "Activity" link should have `aria-current=true`.

<img width="963" alt="Screen Shot 2021-03-18 at 1 41 17 PM" src="https://user-images.githubusercontent.com/38065/111696159-33271700-87f1-11eb-8d5e-f759e11b71fd.png">

**Comment by ianmcburnie** [comment]
Done in 7.4.0
